cmd/nvidia-dra-controller/imex.go

```diff
@@ -312,3 +310,33 @@ func (m *ImexManager) cleanupResourceSlices() error {
 
 	return nil
 }
+
+// getOffset set the offset where an imex domain's channels should start counting from.
+func (i *imexDomainOffsets) getOffset(addedDomainsCh string) (int, error) {
+	var offset int
+	id := strings.Split(addedDomainsCh, ".")
```

Comment:
You should use SplitN and also check that this actually split into two values.

Reply:
We should also create local bariables for `imexDomain` and `cliqueid` instead of always referrring to `id[0]` and `id[1]`.